submissions/sprint_b/app.py

```diff
@@ -0,0 +1,18 @@
+from sprint_b import app
+from sprint_b.helpers import epithet_generator
+from flask import jsonify
```

Comment:
Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
https://www.python.org/dev/peps/pep-0008/#imports